app/views/content/covid_19_guidance.md

```diff
@@ -0,0 +1,54 @@
+* To give you more time to make decisions, we won’t reject or decline any offers automatically until 20 April 2020. Below we outline what this means for you and your candidates. We’ll keep you updated if anything else changes. 
```

Comment:
Not sure why this is a bullet? @laurahermionetennant92

Reply:
It's an arrow on the google doc, I think this got re-translated by my Google-Doc to markdown conversion. Is this meant to be a callout or similar?

Reply:
Hmm. We could use `govuk-inset-text`, but I reckon just having a normal paragraph would be enough. Whatcha think, @laurahermionetennant92?

Reply:
a normal para is fine. i had used the arrow on the google doc to indicate a box out or something similar to highlight the text, but it's not essential. @steventux @paulrobertlloyd